generated/nirfmxspecan/nirfmxspecan.proto

```diff
@@ -5350,12 +5368,12 @@ message IDPDCfgReferenceWaveformRequest {
   double dx = 4;
   repeated nidevice_grpc.NIComplexNumberF32 reference_waveform = 5;
   oneof idle_duration_present_enum {
-    int32 idle_duration_present_raw = 6;
-    IdpdReferenceWaveformIdleDurationPresent idle_duration_present = 8;
+    IdpdReferenceWaveformIdleDurationPresent idle_duration_present = 6;
```

Comment:
Looks like these are also source breaking changes!!!

Reply:
The issue what we are seeing now was fixed in the PR: #1049. This was a permanent fix as we had added the (hard-coded) desired function values in functions_addons.py of the respective personalities. 

But when I used scrapigen to generate the code this time, It is not able to pick up the data from the functions_addons.py file and instead giving the same issue which was reported previously. 

Was there any logical change done on the scrapigen tool? If it is not clear, then we might have to file a bug.
@reckenro

Reply:
Worked through these with the functions_addons.py and __init__.py changes.